package.json

```diff
@@ -11,24 +11,27 @@
   },
   "main": "./dist.js",
   "scripts": {
-    "postinstall": "./node_modules/.bin/webpack",
+    "prepublish": "./node_modules/.bin/webpack --bail",
     "start": "webpack-dev-server --host 0.0.0.0 --content-base .",
     "test": "make test"
   },
-  "dependencies": {
+  "dependencies": {},
+  "devDependencies": {
+    "eslint": "2.7.0",
     "expose-loader": "0.7.1",
+    "gh-pages": "0.11.0",
+    "highlightjs": "8.7.0",
     "htmlparser2": "3.9.0",
     "json-loader": "0.5.4",
+    "lodash.defaultsdeep": "4.6.0",
     "promise": "7.1.1",
-    "webpack": "1.13.0"
-  },
-  "devDependencies": {
-    "eslint": "2.7.0",
-    "highlightjs": "8.7.0",
-    "scratch-blocks": "git+https://git@github.com/LLK/scratch-blocks.git#develop",
-    "scratch-render": "git+https://git@github.com/LLK/scratch-render.git#develop",
+    "scratch-blocks": "0.1.0-80849",
+    "scratch-render": "0.1.0-9f26a",
```

Comment:
They will... although I think if we changed this to `^0.1.0` or just `*`, then we could have the most recent version on npm install.